cfy_manager/components/mgmtworker/mgmtworker.py

```diff
@@ -34,9 +34,8 @@
 
 from ...utils import common
 from ...utils.systemd import systemd
-from ...utils.logrotate import set_logrotate
 from ...utils.install import yum_install, yum_remove
-from ...utils.files import remove_files, deploy, copy_notice, remove_notice
+from ...utils.files import deploy
```

Comment:
I know it's my OCD that I inherited from @nir0s, but if you could maintain the order-by-line-lenth, that would make me sleep better :)

Reply:
Can we migrate to normal, sane, alphabetical order instead? ):)